test/mount_utils_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright Modal Labs 2024
+import pytest
+from collections.abc import Mapping
+
+from modal._utils.mount_utils import validate_mount_points, validate_network_file_systems, validate_volumes
+from modal.exception import InvalidError
+from modal.network_file_system import _NetworkFileSystem
+from modal.volume import _Volume
+
+
+def test_validate_mount_points():
+    # valid mount points
+    dict_input: Mapping = {"/foo/bar": _NetworkFileSystem.from_name("NetworkFileSystem", create_if_missing=False)}
+    validate_mount_points("NetworkFileSystem", dict_input)
+
+    # invalid list input, should be dicts
+    list_input = [_NetworkFileSystem.from_name("NetworkFileSystem", create_if_missing=False)]
```

Comment:
Should the tests exercise the public `NetworkFileSystem` object?

Reply:
I changed that now, these methods seem to only be used for the private objects though, so the code is a bit uglier checking for the public ones too but im fine with it if you prefer it this way!